flask/boxwise_flask/graph_ql/resolvers.py

```diff
@@ -38,41 +44,33 @@ def serialize_date(value):
     return value.isoformat()
 
 
-# registers this fn as a resolver for the "allBases" field, can use it as the
-# resolver for more than one thing by just adding more decorators
-@query.field("allBases")
-def resolve_all_bases(_, info):
-    # discard the first input because it belongs to a root type (Query, Mutation,
-    # Subscription). Otherwise it would be a value returned by a parent resolver.
+@query.field("bases")
+def resolve_bases(_, info):
     return Base.get_all_bases()
 
 
-# not everyone can see all the bases
-# see the comment in https://github.com/boxwise/boxwise-flask/pull/19
-@query.field("orgBases")
-@convert_kwargs_to_snake_case
-def resolve_org_bases(_, info, org_id):
-    response = Base.get_for_organisation(org_id)
-    return response
-
-
 @query.field("base")
 def resolve_base(_, info, id):
     authorization_test("bases", base_id=id)
     response = Base.get_from_id(id)
     return response
 
 
-@query.field("allUsers")
-def resolve_all_users(_, info):
-    response = User.get_all_users()
-    return response
+@query.field("users")
+def resolve_users(_, info):
+    return User.select()
 
 
-# TODO get currrent user based on email in token
 @query.field("user")
 def resolve_user(_, info, email):
-    return get_user_from_email_with_base_ids(email)
+    data = get_user_from_email_with_base_ids(email)
+    data["organisation"] = (
+        Organisation.select()
+        .join(Usergroup)
+        .where(Usergroup.id == data["usergroup"]["id"])
+        .get()
```

Comment:
@aerinsol @vahidbazzaz I just noticed that this lookup does not make much sense anymore. We decided to have the organisation ID sent with the JWT, so here the ID would be used to select the corresponding entry from the organisation table. I have to look into how to test that though.

Reply:
Agreed to work on this in a follow-up PR